### PR TITLE
Add basic tests for the store of @polar/plugin-icon-menu

### PR DESCRIPTION
**Pull request by dopenguin**
## Summary

Add tests for the actions of `@polar/plugin-icon-menu`.

## Instructions for local reproduction and review

`npm run test`

### CODE REVIEW

File: packages/plugins/IconMenu/tests/store.spec.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+          expect(secondParameter.plugin).toEqual('iconMenu')
+
+          expect(dispatch.mock.calls.length).toEqual(0)
+        })
+        it('should do nothing if the client either does not have the same size as the window or the width of the client is considered large', () => {
+          storeModule.actions.openMenuById(actionContext, '')
+
+          expect(commit.mock.calls.length).toEqual(0)
+          expect(dispatch.mock.calls.length).toEqual(0)
+        })
```
Comment (warm-coolguy) [inline packages/plugins/IconMenu/tests/store.spec.ts:77]: This test seems to be identical to the previous test `should do nothing if the menu with the given id is not configured`. Probably a copypaste leftover?

**Review by warm-coolguy — CHANGES_REQUESTED** [review]
🏓 @dopenguin

**Comment by dopenguin** [comment]
> 🏓 @dopenguin

🏓 @warm-coolguy

**Review by warm-coolguy — APPROVED** [review]
>> 🏓 @dopenguin
>>
>🏓 @warm-coolguy 
>

🚀 @dopenguin

**Pull request merged**